service_container/lazy_services.rst

```diff
@@ -124,6 +124,32 @@ laziness, and supports lazy-autowiring of intersection types::
     ) {
     }
 
+Another possibility is to use the :class:`Symfony\\Component\\DependencyInjection\\Attribute\\Lazy` attribute::
+
+    namespace App\Twig;
+
+    use Symfony\Component\DependencyInjection\Attribute\Lazy;
+    use Twig\Extension\ExtensionInterface;
+
+    #[Lazy]
+    class AppExtension implements ExtensionInterface
+    {
+        // ...
+    }
+
+This attribute can be used on a class or on a parameter which should be lazy-loaded, and has a ``lazy`` parameter
```

Comment:
I don't understand this part:

```
and has a ``lazy`` parameter that also supports defining interfaces to proxy and intersection types::
```

What's the purpose of the `lazy` option? (which, by the way, looks like a bad option name to me because the attribute is also called `lazy`)_. Is "type intersection" related to `lazy` option in any way? Thanks!

Reply:
It's a compatibility issue, because the same param exists in both `#[Autoconfigure]` and `#[Autowire]`, and that way the parameter handling is the same.

I admit I'm not sure how to phrase this. But since this is the only parameter, maybe the name can be dropped in the example? Like:
`and has a parameter that also supports defining interfaces to proxy and intersection types::`